Assignment6DRY/src/assignment6dry/Assignment6DRY.java

```diff
@@ -0,0 +1,108 @@
+package assignment6dry;
+
+import java.util.Scanner;
+
+/**
+ * This program lets the user enter address and label sizing information to print
+ * up to 10 labels. 
+ * @author Samantha Shurie
+ */
+public class Assignment6DRY {
+
+    public static void main(String[] args) {
+       
+        Scanner keyboard = new Scanner(System.in);  //accepts input
+        int numLabels;  //stores the number of labels the user wishes to print
+        
+        //allow user to enter the number of labels they would like to print
+        println("Please enter the number of labels you wish to print:");
+        numLabels = keyboard.nextInt();
+        
+        //user can only enter up to 10 labels for this program
+        while(numLabels < 1 || numLabels > 10) {
+            println("Please enter a number between 1 and 10:");
+            numLabels = keyboard.nextInt();
+        }
+        
+        //arrays for storing address and dimension objects
+        Address[] addresses = new Address[numLabels];
+        Dimensions[] dimensions = new Dimensions[numLabels];
+        
+        //until the number of labels is reached, user enters label input
+        for(int i = 0; i < numLabels;i++){
+            inputAddress(addresses, i, keyboard);
+            sizeBox(dimensions, i, keyboard);
+        };
+        
+        //after all data has been entered, print labels
+        for(int i = 0; i < numLabels; i++) {
+            LabelPrinter label = new LabelPrinter(addresses[i], dimensions[i]);
+            label.printLabel();
+        }
+    }
+    
+    /**
+     * Takes a message and prints it to the console
+     * @param message string value
+     */
+    public static void println(String message){
+        System.out.println(message);
+    }
+    
+    /**
+     * Prompts user to enter address data and then assigns the input to an object
+     * in the addresses array
+     * @param addresses array filled with address objects
+     * @param i integer counter variable
+     * @param keyboard Scanner object
+     */
+    public static void inputAddress(Address[] addresses, int i, Scanner keyboard) {
+       //address variables
+        String name;
+	String street;
+	String state;
+	String city;
+	int zip;
+        
```

Comment:
Yes and that is a good practice because in this case it keeps the type as close to the variable as possible. So its easier to infer what the right side of the assignment should return.